ayekan/exporters/deploy-exporters.sh

```diff
@@ -10,25 +10,26 @@ helm repo update
 
 kubectl create ns ${NAMESPACE} --dry-run=client -oyaml | kubectl apply -f -
 
-kubectl apply -f configmap-snmp-config-tmpl.yaml
-kubectl apply -f externalsecret-snmp-community.yaml
-
 # SNMP configuration
-kubectl --namespace ${NAMESPACE} apply -f snmp-configmap.yaml
-kubectl --namespace ${NAMESPACE} apply -f externalsecret-snmp-community.yaml
+kubectl --namespace ${NAMESPACE} apply -f snmp-exporter/configmap.yaml
+kubectl --namespace ${NAMESPACE} apply -f snmp-exporter/xups-configmap.yaml
+kubectl --namespace ${NAMESPACE} apply -f snmp-exporter/schneider-configmap.yaml
+kubectl --namespace ${NAMESPACE} apply -f snmp-exporter/externalsecret.yaml
 helm upgrade --install snmp-exporter prometheus-community/prometheus-snmp-exporter \
      --create-namespace --namespace ${NAMESPACE=} \
      --atomic --timeout 15m \
      --version "1.8.1" \
-     -f ./snmp-exporter.yaml
+     -f ./snmp-exporter/helm-values.yaml
+
+kubectl --namespace ${NAMESPACE} patch deployment snmp-exporter-prometheus-snmp-exporter --patch-file snmp-exporter/volume-patch.yaml
```

Comment:
It looks like the chart has parameters for adding extra volumes: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-snmp-exporter/values.yaml#L61-L65

Would it be possible to chart values instead of a patch?

Reply:
It seems like I missed that in the Helm chart. I though the helm chart didn't allow for direct Kubernetes config to be dropped, but it looks like it actually allows for it: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-snmp-exporter/templates/deployment.yaml#L125

I'll try it out tomorrow.